examples/graphql-documentation/graphql_documentation.md

```diff
@@ -1,8 +1,6 @@
 # GraphQL service - Documentation
 
-A GraphQL schema can include documentation for the schema. These documentation can help you to understand the schema. In Ballerina, the Ballerina doc comments can be used to add documentation to various schema members.
-
-This example shows how to add doc comments to the GraphQL service so that the generated schema will include them as the documentation.
+The Ballerina GraphQL module allows add documentation to the `graphql:Service` and its subsequent types. To add documentation, use the Ballerina doc comments for the `graphql:Service`, `resource`/`remote` methods, types, and `enum`s. Add the documentation to include the descriptions to the generated GraphQL schema.
```

Comment:
```suggestion
The Ballerina GraphQL module allows adding documentation to the `graphql:Service` and its subsequent types. To add documentation, use the Ballerina doc comments for the `graphql:Service`, `resource`/`remote` methods, types, and `enums`. Add the documentation to include the descriptions to the generated GraphQL schema.
```

Reply:
`enum`s should be the correct way right? `enum` is the keyword, not `enums`.